### PR TITLE
Updated dependencies and fixed deprecated getDocument call.

### PR DESCRIPTION
**Pull request by dennybiasiolli**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "grunt": "0.4.5",
+    "babel-core": "^6.5.2",
```
Comment (sayanee) [inline package.json]: Why do we need Babel? Are you using any ES2015 somewhere? Sorry, asking without checking the other code commits.

**Comment by dennybiasiolli** [comment]
Sorry, I forgot babel-core during some tests. Now I removed it.

**Comment by dennybiasiolli** [comment]
Travis CI returns strange error: "Error requesting archive. (...) Make sure your network and proxy settings are correct.", but tests on my machine are all passed. It seems a problem in travis-ci: https://github.com/travis-ci/travis-ci/issues/3225

File: example/js/directives/angular-pdf.js [hunk 2]
```diff
@@ -165,7 +166,7 @@
           }
 
           if (url && url.length) {
-            pdfLoaderTask = PDFJS.getDocument(params, null, null, scope.onProgress);
+            pdfLoaderTask = PDFJS.getDocument(params);
```
Comment (sayanee) [inline example/js/directives/angular-pdf.js]: Hi Thanks for the patch! 

Do you know how else we can refer to `scope.onProgress` if we remove it? See in the [example code](https://github.com/sayanee/angularjs-pdf/blob/be1ab8cf11ddd1c6f2bfb7f7861fb2d758b6f1a5/example/js/controllers/docCtrl.js#L21-L23).

Comment (dennybiasiolli) [inline example/js/directives/angular-pdf.js]: Now is solved in commit 4b0ca8d

**Comment by bastijns-jeroen** [comment]
@sayanee I know you do not maintain this repo anymore, but is it possible to merge this pull request.   That way we can use this library in AngularJs 1.5

**Comment by dennybiasiolli** [comment]
@bastijns-jeroen I'm fixing commits in order to have a clean PR and then we can merge

**Pull request merged**